utility/copy_file.py

```diff
@@ -0,0 +1,40 @@
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class CopyFile:
+    @staticmethod
+    def run(kw, args):
+        """Copy the file from one instance to another.
+
+        Copy the specified file from source to destination instance.
+
+        Usage
+        - copy_file: True
+          source: ceph-rbd1
+          dest: ceph-rbd2
+          path: ./dummy_file
+        """
+
+        cluster = kw["ceph_cluster_dict"][args["source"]]
+
+        for node in cluster:
+            if node.role == "client":
+                source_node = node
+                break
+
+        cmd = f"sleep 10;cat {args['path']}"
```

Comment:
Do we need this? If yes, I think we can handle this from workflow layer

Reply:
Yes, moved it out